tests/get_coverage.yml

```diff
@@ -69,14 +71,17 @@
 - name: Gather coverage data
   hosts: all
   tasks:
-    - shell: "{{ coverage }} combine .coverage.*"
+    - name: gather coverage data
+      command: "{{ coverage }} combine .coverage.*"
```

Comment:
This is wrong because coverage.py does not support wildcards:

```
touch .coverage.test
LANG=C coverage combine ".coverage.*"
Couldn't combine from non-existent path '.coverage.*'
```

Reply:
Thank you, @tyll and @liangwen12year. It was a good lesson!